lib/Frontend/CompilerInvocation.cpp

```diff
@@ -220,44 +223,7 @@ static void updateRuntimeLibraryPaths(SearchPathOptions &SearchPathOpts,
   if (LangOpts.hasFeature(Feature::Embedded))
     LibSubDir = "embedded";
 
-  SearchPathOpts.RuntimeLibraryPaths.clear();
```

Comment:
This is all dead code. It directly touches `SearchPathOpts.RuntimeLibraryPaths`, but then it turns around and calls `SearchPathOpts.setRuntimeLibraryImportPaths` and the end which undoes all of the direct touches.

Reply:
Ah ha, this is why CI is crashing. None of this is dead code. This function is updating `RuntimeLibraryPaths` and `RuntimeLibrary**Import**Paths`. 🤦‍♂️